libmscore/measure.cpp

```diff
@@ -3199,9 +3199,9 @@ Element* Measure::nextElementStaff(int staff)
             auto i = std::find(el().begin(), el().end(), e);
             if (i != el().end()) {
                   if (++i != el().end()) {
-                        Element* e = *i;
-                        if (e)
-                              return e;
+                        Element* el = *i;
```

Comment:
Doesn't that produce any warnings? `el` is the name of the member function which is used just one line above.

Reply:
No,it doesn't (to my surprise)